apiary.apib

```diff
@@ -162,7 +162,25 @@ Update referral system for specific organization
         
 + Response 403 (application/json)        
 
-        {status:0 , msg:"Only organization administrators can update referrals."}        
+        {status:0 , msg:"Only organization administrators can update referrals."}
+        
+### Delete referral system [/organizations/1/referral-system] 
+[DELETE]
+
+Delete referral system for specific organization   
+
++ Request (application/json)
+       {
+           "referral_id": 1
```

Comment:
@hakob9315 I believe it should be `reward_id`

Reply:
@Joran182to As I understand, Hakob means here reward_id, just bad naming, is it right @hakob9315 ?

Reply:
We would need to change this namings to not confuse ourselves